Lunch/CreateAccountViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  CreateAccountViewController.swift
+//  Lunch
+//
+//  Created by Spencer Amann on 8/1/15.
+//  Copyright (c) 2015 Spencer Amann. All rights reserved.
+//
+
+import UIKit
+import Parse
+
+class CreateAccountViewController: UIViewController {
+
+    @IBOutlet weak var userNameTextField: UITextField!
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var confirmedPasswordTextField: UITextField!
+
+    let loginIdentifier = "loginAfterCreateAccount"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    @IBAction func createAccountButtonClick(sender: AnyObject) {
+        let user = PFUser()
+
+        var noErrors = checkkInputForErrors(userNameTextField.text ?? "", password: passwordTextField.text ?? "",
+                                        email: emailTextField.text ?? "", confirmedPassword: confirmedPasswordTextField.text ?? "")
+        if noErrors {
+            user.username = userNameTextField.text
+            user.password = passwordTextField.text
+            user.email = emailTextField.text
+            
+            user.signUpInBackgroundWithBlock { (succes: Bool, error:NSError?) -> Void in
```

Comment:
`succes`

Also I like to make use of type inference and implicit `Void` return type. So it can be shorted to

``` swift
user.signUpInBackgroundWithBlock { (success, error) in
```